addons/MountRoulette/MountRoulette.lua

```diff
@@ -86,4 +102,73 @@ windower.register_event('addon command', function()
     -- Generate random number and use it to choose a mount
     local mount_index = math.ceil(math.random() * #allowed_mounts)
     windower.send_command('input /mount "' .. allowed_mounts[mount_index] .. '"')
+end
+
+commands.blacklist = function(args)
+    local operation = args:remove(1)
+
+    if not operation then
+        windower.add_to_chat(8, 'Blacklisted mounts:')
+        for k, v in ipairs(blacklist) do
```

Comment:
If you make `blacklist` a set, you can use this for more convenient iteration:
```lua
for mount in blacklist:it() do
```

Same below, no need to `ipairs(T(allowed_mounts))`.